src/guides/v2.4/release-notes/b2b-release-notes.md

```diff
@@ -2,3 +2,40 @@
 group: release-notes
 title: B2B Release Notes
 ---
+
+The release notes for the B2B extension captures additions and bug fixes that Magento has added during a release cycle.
+These release notes can include:
+
+-  {:.new}New features
+-  {:.fix}Fixes and improvements
+
+## Magento B2B - Version 2.4.0
+
+-  {:.new} Purchase Orders have been enhanced and rewritten. The are now included my default in Magento Commerce.
```

Comment:
```suggestion
-  {:.new} Purchase Orders have been enhanced and rewritten. The are now included by default in Magento Commerce.
```

Use a variable instead of Magento Commerce. Make sure the Jeanne knows that the feature has been transferred to Commerce.